src/vivarium_dcpn_hypertension_mgt/external_data/drug_efficacy.csv

```diff
@@ -0,0 +1,6 @@
+name,half_dose_efficacy_mean_mean,half_dose_efficacy_mean_lb,half_dose_efficacy_mean_ub,half_dose_efficacy_sd_mean,half_dose_efficacy_sd_sd,standard_dose_efficacy_mean_mean,standard_dose_efficacy_mean_lb,standard_dose_efficacy_mean_ub,standard_dose_efficacy_sd_mean,standard_dose_efficacy_sd_sd,double_dose_efficacy_mean_mean,double_dose_efficacy_mean_lb,double_dose_efficacy_mean_ub,double_dose_efficacy_sd_mean,double_dose_efficacy_sd_sd,half_dose_side_effects_mean,half_dose_side_effects_lb,half_dose_side_effects_ub,standard_dose_side_effects_mean,standard_dose_side_effects_lb,standard_dose_side_effects_ub,double_dose_side_effects_mean,double_dose_side_effects_lb,double_dose_side_effects_ub
+"thiazide_type_diuretics (Diuretics, DU)",7.4,6.6,8.2,0.281,0,8.8,8.3,9.4,0.281,0,10.3,9.4,11.2,0.281,0,2,-2.2,6.3,9.9,6.6,13.2,17.8,11.5,24.2

```

Comment:
are the extra quotes here gonna screw you up ?

Reply:
Nope good catch though. I don't really care about that column anyways. I have separate columns that explicitly code the drug combo - that one is just a human readable one that Meredith put in.